generation/maisi/scripts/diff_model_train.py

```diff
@@ -212,6 +222,7 @@ def train_one_epoch(
         logger (logging.Logger): Logger for logging information.
         local_rank (int): Local rank for distributed training.
         amp (bool): Use automatic mixed precision training.
+        include_body_region (bool): Whether to include body region in data
```

Comment:
Can include_body_region be True when training using rectified flow?

Reply:
It is possible

Reply:
I removed it and make it auto-detect whether to include body_region